releasetools/flashfiles_from_target_files

```diff
@@ -455,6 +455,16 @@ def main(argv):
             for fn, data in configs:
                 ifile = common.File(fn, data)
                 ifile.AddToZip(dest_zip)
+                
+            # Add EFI shell to flashfiles zip
+            efiShellBinaryPath = os.path.join("device","linaro","bootloader","edk2","ShellBinPkg","UefiShell","X64","Shell.efi")
```

Comment:
Will the device-intel-build be used for other non-UEFI platform?
If yes, then need to check only put file for UEFI platform?

Reply:
Made the changes as suggested